psi4/driver/procrouting/empirical_dispersion.py

```diff
@@ -201,7 +201,7 @@ def compute_energy(self, molecule: 'psi4.core.Molecule', wfn: 'psi4.core.Wavefun
                         'level_hint': self.dashlevel,
                         'params_tweaks': self.dashparams,
                         'dashcoeff_supplement': self.dashcoeff_supplement,
-                        'save_pairwise_dispersion': self.save_pairwise_disp,
+                        'pair_resolved': self.save_pairwise_disp,
```

Comment:
Why is this renamed?

Reply:
dftd4 includes a pairwise analysis option directly in its qcschema API under the name `pair_resolved`. since the `save_pairwise_dispersion` option we invented to trigger `-anal` for dftd3 has only been in a couple months, it seemed early enough to drop the d3 name and conform to d4.